app/controllers/issues/states_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Issues::StatesController < AuthenticatedController
+  include ProjectScoped
+
+  def update_states
+    issues = current_project.issues.where(id: params[:ids])
+
+    if issues.update_all(state: state_params, updated_at: Time.now)
+      render json: { state: Issue.state_names[state_params] }
+    else
+      render json: { message: 'Something went wrong.' }
+    end
+  end
+
+  private
+
+  def state_params
+    @state_params ||=
+      if Issue.states.keys.include?(params[:state])
+        params[:state]
+      else
+        :published
```

Comment:
Nothing to be changed here, just curious if there's any reason for the default state to be published

Reply:
@sean-yeoh The reason for this is to eliminate the impact on users who do not want to use Dradis for QA. Since issues are only visible in reports when they are published, users who do not want to use QA would have to always set each issue to published in order to see it in their reports if the default was anything other than published.

Reply:
@MattBudz Got it, thanks for the explanation!